va_explorer/va_logs/models.py

```diff
@@ -1,3 +1,10 @@
 from django.db import models
 
-# Create your models here.
+# import models already created in va data management
+from va_explorer.va_data_management.models import VerbalAutopsy, Location, CauseOfDeath
```

Comment:
Similar to admin.py, are these changes needed? If so, could you expand on what they do for this feature?